lib/index.js

```diff
@@ -681,6 +844,52 @@ MaplibreGeocoder.prototype = {
     return request;
   },
 
+  _getSuggestions: function (searchInput) {
```

Comment:
Not blocking -- Since this is a new API (and I'm unfamiliar with the library), would it be possible to add some unit tests?

Reply:
Yes, @thaddmt is working on some tests for this.

Reply:
Added some unit tests